### PR TITLE
0.6.1

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update version to 0.6.1
- Add patch to run cython and be compatible with py 3.11 (patch taken from editdistance master branch)
- Get source from github (as pypi source is missing a file needed to run cython)

https://github.com/roy-ht/editdistance/tree/v0.6.1
https://anaconda.atlassian.net/browse/PKG-900

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]
LGTM

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  'sha256': '89d016dda04649b2c49e12b34337755a7b612bfd690420edd50ab31787120c1f'
+  url: https://github.com/roy-ht/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: bc58aac72b82bcc22f15d70423990a568c28fda32392fa0eeba66a288e7b5a19
+  patches:
+    # add python 3.11 compat patch: https://github.com/roy-ht/editdistance/commit/5c81574c1ea9538a4d9c97d1f9c28cc10d67980f
+    - patches/0001-cythonize.patch
 
 build:
   number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:16]: Skip py<36, https://github.com/roy-ht/editdistance/blob/cd61f6bff3e0e407f5e744756b45e88c43edd113/setup.py#L40

Comment (cbouss) [reply recipe/meta.yaml:16]: addressed, thank you

**Pull request merged**